pkg/substitution/substitution_test.go

```diff
@@ -340,6 +319,90 @@ func TestValidateEntireVariableProhibitedP(t *testing.T) {
 	}
 }
 
+func TestValidateVariableIsolatedP(t *testing.T) {
```

Comment:
could also update the test names please?

Reply:
done